enhancements/split_helm_ansible_into_their_repos.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Split Helm/Ansible into their own repos
+authors:
+  - "@camilamacedo"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-08-26
+last-updated: 2020-08-30
+status: implementable
+see-also:
+  - "/enhancements/this-other-neat-thing.md"  
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+# Split Helm/Ansible into their projects
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions 
+
+For Helm, besides the requirement of split the repo it also has a new pkg implementation which requires to be putted in place and is in the [joelanford/helm-operator](https://github.com/joelanford/helm-operator) currently. 
+
+This proposal is decoupling both requirements, which allow a specific strategy for each one of them. However, would this option be more comfortable to move forward with both needs or the approach to address them for Helm simultaneously could be more practical?
+
+**Note** A enhancement to describe the possible options to address the Helm requirement for the new pkg implementation ought to be made.
```

Comment:
It was done. See: https://github.com/operator-framework/enhancements/pull/45/files
If that gets merged than we can update this doc with a link for this proposal.